src/components/ForgetPassword.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState,useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { Typography } from '@mui/material';
+import { ToastContainer } from 'react-toastify';
+import Loading from './Loading';
+import { requestPasswordReset,reset } from '../states/features/auth/authSlice';
+
+import { ErrorNotification, successNotification } from './Notification';
+
+
+
+const ForgetPassword = () => {
+  const [email, setEmail] = useState('');
+  
+  const { register, handleSubmit, } = useForm();
+  const navigate=useNavigate()
+
+  const handleEmailChange = (e) => {
+    setEmail(e.target.value);
+  };
+
+  const dispatch = useDispatch();
+  const { isLoading,isError,isSuccess, message } = useSelector((state) => {
+    return state.auth;
+  });
+
+  
+
+
+const onSubmit = () => {
+  if (!email) {
+    message('Please enter your email address.');
```

Comment:
This section should be removed because it is redundant when using react-hook-form. With react-hook-form, you cannot submit the form if the email is not valid, and any errors will be displayed automatically.